qml/keys/AnnotatedKey.qml

```diff
@@ -128,42 +129,31 @@ Item {
 
     Text {
         id: annotationLabel
-        //text: (UI.currentShiftState != "NORMAL") ? __annotationLabelShifted : __annotationLabelNormal
         text: __annotationLabelNormal
 
-        anchors.left: useHorizontalLayout ? parent.left : undefined
-        anchors.leftMargin: useHorizontalLayout ? Units.gu(2.0) : 0
-
-        anchors.horizontalCenter: useHorizontalLayout ? undefined : buttonImage.horizontalCenter
-        anchors.horizontalCenterOffset: useHorizontalLayout ? Units.gu(2.75) : 0
+        anchors.horizontalCenter: buttonImage.horizontalCenter
+        anchors.horizontalCenterOffset: useHorizontalLayout ? UI.keyWidth/-6 : 0
 
         anchors.verticalCenter: parent.verticalCenter
-        anchors.verticalCenterOffset: useHorizontalLayout ? Units.gu(-0.25) : Units.gu(-1.75)
+        anchors.verticalCenterOffset: useHorizontalLayout ? Units.gu(-0.25) : UI.formFactor === "phone" && UI.isLandscape ? Units.gu (-1.00) : Units.gu(-1.75) 
```

Comment:
maybe we should use keyHeight for that sort of precise layout ? like you used keyWidth just above, which is pretty smart imho